DataFormats/PatCandidates/src/TauPFEssential.cc

```diff
@@ -15,6 +15,7 @@ pat::tau::TauPFEssential::TauPFEssential(const reco::PFTau& tau) :
     hcalEnergy_(0.),
     leadingTrackNormChi2_(1.e+3),
     etaAtEcalEntrance_(0.),
+    phiAtEcalEntrance_(0.),
```

Comment:
You can leave this stuff in but I would not add it to the next PR. We take this one on a separate note.